org.adempiere.base/src/org/adempiere/model/CalloutPrintFormatType.java

```diff
@@ -0,0 +1,29 @@
+package org.adempiere.model;
+
+import java.util.Properties;
+
+import org.adempiere.base.IColumnCallout;
+import org.compiere.model.GridField;
+import org.compiere.model.GridTab;
+import org.compiere.model.X_AD_PrintFormatItem;
+import org.compiere.util.DB;
+
+public class CalloutPrintFormatType implements IColumnCallout {
+
+	@Override
+	public String start(Properties ctx, int WindowNo, GridTab mTab, GridField mField, Object value, Object oldValue) {
+
+		String printFormatType = (String)value;
+
+		if(printFormatType == null || printFormatType.isEmpty() || !printFormatType.equals(X_AD_PrintFormatItem.PRINTFORMATTYPE_Script))
```

Comment:
this can actually be simplify to just !(X_AD_PrintFormatItem.PRINTFORMATTYPE_Script.equals(printFormatType))

Reply:
Fixed